ydb/core/tx/schemeshard/schemeshard__operation_common_subdomain.h

```diff
@@ -6,33 +6,43 @@
 namespace NKikimr {
 namespace NSchemeShard {
 
-inline bool CheckStorageQuotasKinds(const Ydb::Cms::DatabaseQuotas& quotas,
-                                    const TVector<TStoragePool>& pools,
-                                    const TString& path,
-                                    TString& error
+inline bool CheckStoragePoolsInQuotas(const Ydb::Cms::DatabaseQuotas& quotas,
+                                      const TVector<TStoragePool>& pools,
+                                      const TString& path,
+                                      TString& error
```

Comment:
No need to align args to the opening brace, double indentation from the left is enough.

Reply:
I hope I got your idea right